deployment/terraform/variables.tf

```diff
@@ -0,0 +1,221 @@
+variable "project" {
+  default = "Flood Map"
+  type    = string
+}
+
+variable "environment" {
+  default = "Staging"
+  type    = string
+}
+
+variable "aws_region" {
+  default = "us-east-1"
+  type    = string
+}
+
+variable "aws_availability_zones" {
+  default = ["us-east-1a", "us-east-1b"]
+  type    = list(string)
+}
+
+variable "aws_key_name" {
+  type = string
+}
+
+variable "r53_private_hosted_zone" {
+  type = string
+}
+
+variable "vpc_cidr_block" {
+  default = "10.0.0.0/16"
+  type    = string
+}
+
+variable "external_access_cidr_block" {
+  type = string
+}
+
+variable "vpc_private_subnet_cidr_blocks" {
+  default = ["10.0.1.0/24", "10.0.3.0/24"]
+  type    = list(string)
+}
+
+variable "vpc_public_subnet_cidr_blocks" {
+  default = ["10.0.0.0/24", "10.0.2.0/24"]
+  type    = list(string)
+}
+
+variable "bastion_ami" {
+  type = string
+}
+
+variable "bastion_instance_type" {
+  default = "t3.nano"
+  type    = string
+}
+
+variable "bastion_ebs_optimized" {
+  default = true
+  type    = bool
+}
+
+variable "rds_allocated_storage" {
+  default = 32
+  type    = number
+}
+
+variable "rds_engine_version" {
+  default = 11
```

Comment:
I think that we should aim for PostgreSQL 12 here. This will likely necessitate updating the PostgreSQL client libraries on the bastion.

Reply:
RDS instance has been updated and the Postgres 12 client libraries has been installed on the bastion. `amazon-linux-extras` doesn't have 12 yet so I installed via [Postgres' provided RPMs](https://download.postgresql.org/pub/repos/yum/12/redhat/rhel-7-x86_64/).

Reply:
Were the RPMs installed in a way that allow the PostgreSQL packages to be upgraded via something like `yum update`? If not, I think the process you used to install it may be worth noting in this PR somewhere because that may become important if/when we need to upgrade in the future.

With RF, I ended up installing PostgreSQL client libraries from source because the PGDG RPMs didn't apply cleanly. Compiling from source isn't ideal, but knowing things were compiled from source helps make sense of the path to future upgrades.

Reply:
The `postgresql12` package can be managed by Yum. I don't think it will detect updates since the RPM was installed individually, rather than as part of a larger upstream repo.

If we need to upgrade in the future, I would hope that the `postgresql12` library would be available in `amazon-linux-extras` since RDS now supports 12. I've seen AWS support threads acknowledging this need but it seems to be in earlier stages of adoption.

I'll add a note to a README that details how it was installed and provides a link to where I retrieved the RPM.